### PR TITLE
test-kontrol: optimize the doc

### PR DESCRIPTION
**Pull request by rickck11**
Make the document more neutral to reduce misunderstandings.

Similar to https://github.com/ethereum-optimism/optimism/pull/12054, DeploymentSummary.t.sol has been removed, using it as an example in this document here would be very confusing.

### CODE REVIEW

**Comment by tynes** [comment]
@smartcontracts Is this right?

File: packages/contracts-bedrock/test/kontrol/README.md [hunk 1]
```diff
@@ -148,7 +142,7 @@ function setUp() public {
     superchainConfig = SuperchainConfig(superchainConfigProxyAddress);
 }
 ```
-Note that the names of the addresses come from [`DeploymentSummary.t.sol`](deployment/DeploymentSummary.t.sol) and are automatically generated by the [`make-summary-deployment.sh`](./scripts/make-summary-deployment.sh) script.
+Note that the names of the addresses automatically generated by the [`make-summary-deployment.sh`](./scripts/make-summary-deployment.sh) script.
```
Comment (AmadiMichael) [inline packages/contracts-bedrock/test/kontrol/README.md]: nit: 
```suggestion
Note that the names of the addresses are automatically generated by the [`make-summary-deployment.sh`](./scripts/make-summary-deployment.sh) script.
```

Comment (rickck11) [reply packages/contracts-bedrock/test/kontrol/README.md]: using 'are' is better than 'is' here? @AmadiMichael

Comment (AmadiMichael) [reply packages/contracts-bedrock/test/kontrol/README.md]: Yes, true

**Review by smartcontracts — APPROVED** [review]

**Comment by smartcontracts** [comment]
Hi, apologies for the delay here, would you mind rebasing this PR?

**Comment by rickck11** [comment]
@smartcontracts rebased, thanks!

**Comment by smartcontracts** [comment]
Thanks! Merging :-)

**Pull request merged**